flipkart_scraper.py

```diff
@@ -0,0 +1,140 @@
+import requests  
+from urllib.request import urlopen as ureq
+from bs4 import BeautifulSoup as soup 
+import smtplib 
+import io  
+import csv 
+# Call the total script like : scrape_page_type_1(get_urls(4,6),6), where 4 specifies the dropdownselection and 6 is the number of pages
+
+# The front end can make use of the passing parameters
+# In the dropdown list, the developer can give the Words like Laptops, Mobiles, Cameras, Headphones, Air conditioners etc
+# Also the user can define the limit of number of pages to scrap [It is used in both functions]
+
+def get_urls(DropDownSelection,limit):
+  '''When get_url() is called, it makes a list of links according to the limit and selection
+  in order to forward to the scraper part
+  Argument1= DropDownSelection :: Selects the category from the frontend
+  Argument2= limit :: The limit for the number of web pages selected by user or by developer in frontend'''
+  urlss=[] 
+  # 1 = Laptops, 2 = Mobiles, 3 = Cameras, 4 = Headphones, 5 = AC's  
+  switch= {
+      1: 'https://www.flipkart.com/laptops/pr?sid=6bo,b5g&otracker=categorytree&page=',
+      2: 'https://www.flipkart.com/mobiles/pr?sid=tyy%2C4io&otracker=categorytree&page=',
+      3: 'https://www.flipkart.com/cameras/pr?sid=jek,p31&otracker=categorytree&page=',
+      4: 'https://www.flipkart.com/headphones/pr?sid=fcn&otracker=categorytree&page=',
+      5: 'https://www.flipkart.com/air-conditioners/pr?sid=c54&otracker=categorytree&page='
+  }
+  for i in range(1,limit):
+    item = switch[DropDownSelection] + str(i)
+    urlss=urlss+[item]
+  # Change the variable with a dropdown box when merging with its front end according to the need
+  '''return(list)::
+  Returns the list names urlss that contains the list of links based on the given category'''
```

Comment:
Docstrings are only used to describe classes, functions etc. Please remove this one.

Reply:
Docstrings aren't used here, please remove this and refer to the article I shared earlier